content/docs/1.5.0/deploy/upgrade/_index.md

```diff
@@ -9,6 +9,26 @@ Here we cover how to upgrade to the latest Longhorn from all previous releases.
 
 There are no deprecated or incompatible changes introduced in v{{< current-version >}}.
 
+# Upgrade Path Enforcement
+
+Since Longhorn v1.5.0, we enforce an upgrade path. Enforce an upgrade path to prevent users from upgrading from any unsupported version. After rejecting the user's upgrade attempt, the user's Longhorn setup should remain intact without any impacts.
+
+Compare current Longhorn version with upgrading/downgrading Longhorn version, only allow authorized version upgrade (e.g., 1.3.x to 1.5.x is not allowed) as following table.
+
+  |  Current version |  Upgrading version |  Allow |
+  |    :-:      |    :-:      |   :-:  |
+  |  x.y.*      |  x.(y+1).*  |   ✓    |
+  |  x.y.0      |  x.y.*      |   ✓    |
+  |  x.y.*      |  (x+1).y.*  |   ✓    |
```

Comment:
We should only the upgrading from a last patch version of the just previous release branch to a new major version.

1.30.* to 2.0.* given 1.30 is the last feature branch of major 1.

Reply:
Added a footnote to mention about the last feature version of previous release branch. Thanks